documentation/content/en/books/handbook/wine/_index.adoc

```diff
@@ -52,7 +52,7 @@ endif::[]
 == Synopsis
 
 https://www.winehq.org/[WINE], which stands for Wine Is Not an Emulator, is technically a software translation layer.
-It enables to install and run some software written for Windows(R) on FreeBSD (and other) systems.
+It enables to install and run software written for Windows(R) on FreeBSD (and other) systems.
```

Comment:
I fixed this with a follow-up commit since it was a preexisting issue unaffected by your edit, but "enables" is a transitive verb that needs a direct object, like "it enable users to install and run software" ... but "users" is pretty vague in this context so I went with an alternative verb that is intransitive.

Reply:
Thanks for the update!